code/generate_setting/generate_setting.R

```diff
@@ -0,0 +1,20 @@
+args = commandArgs(trailingOnly=FALSE)
+print(args[4])
+setting_file = paste(gsub(".R$", "", gsub("--file=", "", args[4])), ".csv", sep="")
+print(setting_file)
+
+setting = list(
+               random_seed = c(51:100),
```

Comment:
Add comment for what the numbers represent (for all fields)

Reply:
added.